std/stdio.d

```diff
@@ -4683,18 +4683,20 @@ alias stdin = makeGlobal!(StdFileHandle.stdin);
 @safe unittest
 {
     // Read stdin, sort lines, write to stdout
-    import std.algorithm.mutation : copy;
     import std.algorithm.sorting : sort;
     import std.array : array;
+    import std.range.primitives : put;
     import std.typecons : Yes;
 
     void main() {
-        stdin                       // read from stdin
-        .byLineCopy(Yes.keepTerminator) // copying each line
-        .array()                    // convert to array of lines
-        .sort()                     // sort the lines
-        .copy(                      // copy output of .sort to an OutputRange
-            stdout.lockingTextWriter()); // the OutputRange
+        auto output = stdout.lockingTextWriter();
+        put(
+            output,                             // writing directly to stdout
+            stdin                               // read from stdin
+                .byLineCopy(Yes.keepTerminator) // copying each line
+                .array()                        // convert to array of lines
+                .sort()                         // sort the lines
```

Comment:
I'm going to repeat @CyberShadow's [comment](https://github.com/dlang/phobos/pull/6039#discussion_r161854016) as it got hidden with the last rebase:

> Boo, this breaks the UFCS chain. Is there as UFCS-able `put` in Phobos?

Reply:
Yah, that's a problem.

Reply:
This is an existential problem with `put`, because the member hook is also called `put`.